api/debug.go

```diff
@@ -361,6 +361,24 @@ func (d *DebugAPI) TraceCall(
 	return tracer.GetResult()
 }
 
+// FlowHeight returns the Flow height for the given EVM block.
+func (d *DebugAPI) FlowHeight(
```

Comment:
```suggestion
func (d *DebugAPI) FlowHeightByEVMBlock(
```

Reply:
Then we would have to query it by: `eth_flowHeightByEVMBlock`, it's rather easy to get it wrong :sweat_smile:

Reply:
>Then we would have to query it by: eth_flowHeightByEVMBlock, it's rather easy to get it wrong 😅

Why it's easy to get wrong? Sorry, I didn't get it.

Reply:
I see. How do we map a DebugAPI method to a RPC method. Is there a conversion defined somewhere? Or could be specify `FlowHeightByEVMBlock` API will be mapped to `eth_flow_height_by_evm_block` something like that?

Reply:
Unfortunately, this isn't something that we have control over, it's specific by some reflection that the JSON-RPC server from Geth does.

Reply:
I was actually wrong, it will be `debug_flowHeightByEVMBlock` and not `eth_*`.

Reply:
OK. Is `TraceBlockByNumber` auto mapped to `debug_traceBlockByNumber`? If so, `debug_flowHeightByEVMBlock` is the same naming convention, right?

Reply:
Yep, the naming convention is just to prepend the namespace (e.g. `eth_` / `net` / `debug`) and lowercase the first character of the method name. `debug_flowHeightByEVMBlock` is also fine, I believe we will have this command documented, so it's not like we will write it by hand.

Reply:
Though I am not entirely sure if it will be `debug_flowHeightByEVMBlock` or `debug_flowHeightByEvmBlock`.

Reply:
>Though I am not entirely sure if it will be debug_flowHeightByEVMBlock or debug_flowHeightByEvmBlock.

Oh, I see your concern now. Yeah, let's find out.

I'm ok that we name it to `FlowHeightByBlock` to avoid being mapped to `debug_flowHeightByEvmBlock`, which looks weird. In that case, we better mention EVM block in the comments.

Reply:
I tried this locally, and it will be mapped to `debug_flowHeightByEVMBlock`, so only the first character of the method name gets lower-cased. The rest remains intact.